types/bootstrap/js/dist/alert.d.ts

```diff
@@ -1,18 +1,13 @@
-declare class Alert {
-    constructor(element: Element);
+import { BaseClass } from './common';
```

Comment:
IMO there is no `common`, everything resides in same directory, so relative . imports